jobs/build_docker/prepare_docker_post_test.sh

```diff
@@ -61,3 +61,25 @@ set -e
 
 docker-compose -f docker-compose-mini.yml up > $WORKSPACE/build-log/vagrant.log &
 popd
+
+#"common" folder is the deepest folder in mount folder which is used to share files on docker
```

Comment:
in your comments, can you point this to RackHD/docker/docker-compose*.yml
because this info is hidden in that file.